test/test_simulation_results.jl

```diff
@@ -7,19 +7,35 @@ function verify_export_results(results, export_path)
 
     for problem_results in values(results.problem_results)
         rpath = problem_results.results_output_folder
+        problem = problem_results.problem
         for timestamp in get_timestamps(problem_results)
             for name in list_dual_names(problem_results)
-                compare_results(rpath, export_path, model, "duals", name, timestamp)
+                @test compare_results(rpath, export_path, problem, "duals", name, timestamp)
             end
             for name in list_parameter_names(problem_results)
-                compare_results(rpath, export_path, model, "parameters", name, timestamp)
+                @test compare_results(
+                    rpath,
+                    export_path,
+                    problem,
+                    "parameters",
+                    name,
+                    timestamp,
+                )
             end
             for name in list_variable_names(problem_results)
-                compare_results(rpath, export_path, model, "variables", name, timestamp)
-            end
-            for name in list_aux_variable_names(problem_results)
-                compare_results(rpath, export_path, model, "aux_variables", name, timestamp)
+                @test compare_results(
+                    rpath,
+                    export_path,
+                    problem,
+                    "variables",
+                    name,
+                    timestamp,
+                )
             end
+            # TODO: These aren't stored yet.
```

Comment:
@daniel-thom a simulation with StandardUnitCommitment should have auxiliary variables to test this.